packages/sitecore-jss-angular/src/components/base-field.directive.spec.ts

```diff
@@ -162,8 +160,95 @@ describe('<span *scTestBase />', () => {
         comp.editable = false;
         fixture.detectChanges();
 
-        const rendered = deSpan.nativeElement.innerHTML;
-        expect(rendered).toBe('');
+        const spanElement = de.query(By.css('span'));
+        expect(spanElement).toBeNull();
+      });
+    });
+
+    describe('with "metadata" property value', () => {
+      describe('and editing enabled', () => {
+        it('should render <img /> with metadata chrome tags', () => {
+          const field = {
+            metadata: testMetadata,
+            value: 'value',
+          };
+          comp.editable = true;
+          comp.field = field;
+          fixture.detectChanges();
+
+          const fieldValue = de.query(By.css('span'));
+          const metadataOpenTag = fieldValue.nativeElement.previousElementSibling;
+          const metadataCloseTag = fieldValue.nativeElement.nextElementSibling;
+
+          expect(metadataOpenTag.outerHTML).toEqual(
+            `<code scfieldmetadatamarker="" type="text/sitecore" chrometype="field" kind="open" class="scpm">${JSON.stringify(
+              testMetadata
+            )}</code>`
+          );
+          expect(metadataCloseTag.outerHTML).toEqual(
+            '<code scfieldmetadatamarker="" type="text/sitecore" chrometype="field" kind="close" class="scpm"></code>'
+          );
+        });
+
+        it('should render empty field with metadata chrome tags', () => {
+          const field = {
+            metadata: testMetadata,
+            value: '',
+          };
+          comp.editable = true;
+          comp.field = field;
+          fixture.detectChanges();
+
+          const fieldValue = de.query(By.css('sc-default-empty-text-field-editing-placeholder'));
```

Comment:
Please add an assert for the empty field span being present

Reply:
I don't think it should be asserted here as this tests checks a single behavior, that metadata wrappers are still present. There are other tests that field produces expected response for empty values.